doc/modules/language-guide/pages/language-manual.adoc

```diff
@@ -249,16 +249,16 @@ For example, type `Int` is both arithmetic (A) and ordered (O) and supports both
 
 |===
 | `<relop>` | Category |
-|  `==` |  | equals
-|  `!=` |  | not equals
-| `␣<␣` | O | less than *(must be enclosed in whitespace)*
-| `␣>␣` | O | greater than *(must be enclosed in whitespace)*
-|  `\<=` | O | less than or equal
-|  `>=` | O | greater than or equal
+|  `+==+` |  | equals
+|  `+!=+` |  | not equals
+| `␣<␣` | O | less than _(must be enclosed in whitespace)_
+| `␣>␣` | O | greater than _(must be enclosed in whitespace)_
```

Comment:
Why do we have `␣`s here and in the prev line?

Reply:
As a way to indicate that the spaces are required. I'm open to better suggestions...